### DISCUSSION TITLE
Issue Launching Specific Steam Games

### DISCUSSION THREAD

**Discussion by FlyE32** [hyprwm/Hyprland#8711] [Q&A]
Good day all,

Been having a few issues with launching certain steam titles in Hyprland. Tested All titles in KDE and Gnome as a sanity check to see if it was potential compatibility tool issue (though all games are linux native). Alas, they worked with no hangup on those DEs.

Said games in question are Baldur's Gate 3 (BG3) & Balatro.

BG3 will bring me to the launcher. If i select Vulkan, I will receive an error; "failed to collect displays". If launched in DX11, I receive audio, yet no output.
![image](https://github.com/user-attachments/assets/4fef40f2-c08e-493b-be0d-e47af6f2cb0d)

Balatro will not display at all when launched.

------------

Kind of wish I knew where to start with this trouble shooting process, as again, these work just fine on both KDE and Gnome. Any help would be greatly appreciated!

**Comment by izmyname** [comment]
You'd better make an issue, specify what distro you're using and post your config. It's near impossible to say - what's wrong, without any useful information.

**Reply by FlyE32** [reply]
Sorry for just now getting back to you. The problem was related to a broken pypr script that wasn't grabbing monitor data. Hence the failed to collect displays message.

**Reply by yotov96** [reply]
Hey I'm running into the exact same issue with these 2 games, can you share how you managed to fix it if you did?

**Reply by AlienTux** [reply]
@FlyE32 I would also be interested in this solution please.

@yotov96 in the meantime you can use Gamescope to run the game:
`LD_PRELOAD="" gamescope -W 1920 -H 1080 -f -e --  %command% --skip-launcher --vulkan`

I added the `LD_PRELOAD=""` because after playing for a while the game starts to stutter and I found this solution online. Haven't tested it enough to say it worked tho. Also, remember to change the `-W` and `-H` flags to suit your monitors resolution.
